OpenSim/Simulation/SimbodyEngine/Joint.h

```diff
@@ -267,27 +253,36 @@ OpenSim_DECLARE_ABSTRACT_OBJECT(Joint, ModelComponent);
     */
     virtual void scale(const ScaleSet& aScaleSet);
 
-    /**
-    * ModelComponent display interface.
-    *
-    * This method appends the visuals for the Joint to the list appendToThis.
-    * Base class adds geometry for the two Frames making the joint, any extra 
-    * gometry needed to visualize the Joint would be added by the subclass.
+protected:
+    /** A Coordinate place-holder created by constructCoordinate(). E.g.:  
+    class My2DofJoint::Joint {
+        CoordinateP dof1{ constructCoordinate(Coordinate::MotionType::Rotational) };
+        CoordinateP dof2{ constructCoordinate(Coordinate::MotionType::Translatinal) };
+        ...
```

Comment:
Are these guaranteed to construct in the order you list? I think this is worth mentioning in the comment. This could be another reason to use an initializer list.